examples/nodejs/index.html

```diff
@@ -0,0 +1,7 @@
+<script type="text/javascript">
+const url = new URL('http://localhost/subscribe');
+url.searchParams.append('topic', 'https://example.com/books/{id}');
+
+const eventSource = new EventSource(url);
```

Comment:
Can you use the discovery mechanism? (available in the readme)

Reply:
I won't be able to run the fetch `example.com/books` though :|. What I had in mind was a simple example that you can run on your device to see the magic happened.

Reply:
Yes It's why I used localhost in the first time. I think we should replace `example.com` by `localhost` everywhere.